modules/python/src2/gen2.py

```diff
@@ -182,6 +204,25 @@
     }
 """)
 
+stub_header = """import numpy as np
+
+from typing import (
+    Any,
+    overload,
+    Sequence,
+    Tuple,
+)
+
+"""
+
+stub_type_aliases = {
+    "Mat": "np.ndarray",
+    "Rect": "Sequence[int]",
+    "Size": "Sequence[int]",
+    "Size2f": "Sequence[float]",
+    "UMat": "np.ndarray"
```

Comment:
There is an existing class in OpenCV Python -> `cv2.UMat` so this type alias is redundant.